platform/broadcom/sonic-platform-modules-delta/debian/platform-modules-et-6248brb.init

```diff
@@ -14,6 +14,14 @@
 case "$1" in
 start)
     echo -n "Setting up board... "
+    if [ `grep -c usemsi /etc/init.d/opennsl-modules` = "0" ]; then
+        sed -i "s/debug=4 dma_debug=1/debug=4 dma_debug=1 usemsi=1/g" /etc/init.d/opennsl-modules
```

Comment:
@judyjoseph , can you comment on this?

Reply:
Please refer to the updated Porting guide (https://github.com/Azure/SONiC/wiki/Porting-Guide) on how to use the **platform_env.conf** file to override the parameters to platform modules.  You could create this file in the <onie_platform_string>DIR , check the presence of this file in your init script and use the <key> <value> defined there in the file.

Reply:
I removed "usemsi=1" from init file to ./mellanox/x86_64-mlnx_msn2740-r0/platform_reboot